README.md

```diff
@@ -1421,6 +1421,36 @@ condition](#safe-assignment-in-condition).
   you forget either of the rules above!
 <sup>[[link](#always-warn-at-runtime)]</sup>
 
+* <a name="no-nested-methods"></a>
+  Do not use nested method definitions, use lambda instead.
+  Nested method definition is equal to the definition in the same scope as
```

Comment:
Nested method definitions actually produce methods in the same scope (e.g. class) as the out method. Furthermore, the "nested method" will be redefined every time the method containing its definition is invoked.